Traceback (most recent call last):
  File "c:\Users\gizem\Documents\GitHub\noisy_fair\mixture_outlier.py", line 669, in <module>
    aspect_ratio_l1(d=100, n=100, change_d=False, n_runs=2, s=1, SNR=10, mixture=True, l2=False,outlier_strength=100)
  File "c:\Users\gizem\Documents\GitHub\noisy_fair\mixture_outlier.py", line 442, in aspect_ratio_l1
    err_mm, errs_avm_poly, err_train_mm, errs_train_avm_poly, err_mm_l1, errs_avm_poly_l1, err_train_mm_l1, errs_train_avm_poly_l1,  wmm_l2, w_l2, wmm_l1, w_l1 = margin_classifiers_perf(d=d,n=n,approx_tau=1, SNR=SNR, n_test=1e4, s=s, l1=True, l2=l2, random_flip_prob=sigma, mixture=mixture, outlier_strength=outlier_strength)
  File "c:\Users\gizem\Documents\GitHub\noisy_fair\mixture_outlier.py", line 239, in margin_classifiers_perf
    xs, ys, x_seq_test, y_seq_test, z1s, z2s = create_data_mixture(d,n1,n2,n_test, s=s, random_flip_prob=random_flip_prob, SNR=SNR, outlier_strength=outlier_strength)
TypeError: create_data_mixture() got an unexpected keyword argument 'outlier_strength'